hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -758,9 +763,8 @@ static int xpadneo_input_mapping(struct hid_device *hdev, struct hid_input *hi,
 	int i = 0;
 
 	/* XBE2 reports a full keyboard, which we don't need */
-	if ((usage->hid & HID_USAGE_PAGE) == HID_UP_KEYBOARD) {
+	if ((usage->hid & HID_USAGE_PAGE) == HID_UP_KEYBOARD)
```

Comment:
This may actually be part of chatpad support but I'm not sure if that is even supposed to work via Bluetooth. Unless someone has this hardware, we can't test it, and it should stay disabled as it is now.